exchanges/gateio/gateio_wrapper.go

```diff
@@ -2161,25 +2151,22 @@ func (g *Gateio) GetFuturesContractDetails(ctx context.Context, item asset.Item)
 			case "BI-QUARTERLY":
 				ct = futures.HalfYearly
 				s = e.Add(-kline.SixMonth.Duration())
-			default:
-				ct = futures.LongDated
 			}
-			contractsToAdd[j] = futures.Contract{
+			resp[i] = futures.Contract{
 				Exchange:             g.Name,
 				Name:                 name,
 				Underlying:           underlying,
 				Asset:                item,
 				StartDate:            s,
 				EndDate:              e,
 				SettlementType:       futures.Linear,
-				IsActive:             !contracts[j].InDelisting,
+				IsActive:             !contracts[i].InDelisting,
 				Type:                 ct,
 				SettlementCurrencies: currency.Currencies{currency.USDT},
 				MarginCurrency:       currency.Code{},
-				Multiplier:           contracts[j].QuantoMultiplier.Float64(),
-				MaxLeverage:          contracts[j].LeverageMax.Float64(),
+				Multiplier:           contracts[i].QuantoMultiplier.Float64(),
+				MaxLeverage:          contracts[i].LeverageMax.Float64(),
 			}
-			resp = append(resp, contractsToAdd...)
```

Comment:
https://www.youtube.com/watch?v=3w-2gUSus34
Use of `j` and this shows some missed copy paste action